LaunchUpCorp/Focus-Finder
# feat(FOC-10): set up postgres with prisma

## feat(FOC-10): set up postgres with prisma

### Type of Change


### Description

- Set up Postgres db with Prisma and connected Graphql apis to make data changes persistent in database.
- Dependencies required for this change
  - Prisma
  - @prisma/client
  - Postgres installed and running on the machine

### Testing Instructions

- Make sure PostgreSQL is installed and running on your machine. (See docs for installation: https://www.postgresql.org/download/)
- Create .env file and copy contents in .env.example and update values to your own.
- Run `npm install` to install dependencies.
- Run `npx prisma migrate dev` to apply migrations. (To learn more about Prisma, visit: https://www.prisma.io/docs)
- Run `npm run start`(development) or `npm run start:dev`(watch mode) to run server on `localhost:3000`.
- Visit `localhost:3000/graphql` to test APIs in the playground.

### Checklist (delete irrelevant items)

- [x] I have followed coding standards and best practices.
- [x] I have updated the relevant tests if needed.
